modules/manage/partials/authentication.adoc

```diff
@@ -687,7 +690,43 @@ pandaproxy_client:
 ----
 endif::[]
 
+[[plain]]
+==== SASL/PLAIN
+
+You can configure Kafka clients to authenticate using either SASL/SCRAM or SASL/PLAIN with a
+single account using the same username and password. Unlike SASL/SCRAM, which uses a challenge response with hashed
+credentials, SASL/PLAIN stores plaintext passwords, but requires TLS for external encryption.
+
+If you have existing PLAIN Kafka clients and applications, you can migrate to Redpanda without updating
+your application by creating local Redpanda SCRAM accounts and enabling PLAIN as a authentication mechanism.
+
+NOTE: Clusters configured with _only_ a SASL/PLAIN mechanism are not supported.
+
+===== Enable SASL/PLAIN
+
+You must enable SASL/PLAIN  explicitly by appending PLAIN to the list of SASL mechanisms:
+
+[,bash]
+----
+rpk cluster config set sasl_mechanisms '["PLAIN"]'
+----
+
+To enable SASL/PLAIN authentication for the Kafka API, set the `authentication_method` property of the Kafka listeners to `sasl_plain`.
+
+In `redpanda.yaml`, enter:
+
+[,yaml,lines=6]
+----
+redpanda:
+  kafka_api:
+    - address: 0.0.0.0
+      port: 9092
+      name: sasl_plain_listener
+      authentication_method: sasl_plain
```

Comment:
We specify `sasl`, the allowed `sasl` mechanisms ares set with `sasl_mechanisms`
```suggestion
      authentication_method: sasl
```